http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -118,18 +118,22 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
             runtimeToSave = SaveCluster(cluster = runtime, runtimeConfig = runtimeConfig, now = ctx.now)
             savedRuntime <- clusterQuery.save(runtimeToSave).transaction
 
-            task = Task(
+            task = for {
+              _ <- createRuntime(CreateAzureRuntimeParams(workspaceId, savedRuntime, runtimeConfig, disk, runtimeImage),
+                                 WsmJobControl(createVmJobId))
+              _ <- publisherQueue.offer(
```

Comment:
if this is the change that's causing unit tests to hang, I would just leave things as is cuz it'll be fixed by a PR I'll have soon